src/main/sensors/gyro.h

```diff
@@ -155,9 +154,12 @@ typedef enum {
     YAW_SPIN_RECOVERY_AUTO
 } yawSpinRecoveryMode_e;
 
-#define GYRO_CONFIG_USE_GYRO_1      0
-#define GYRO_CONFIG_USE_GYRO_2      1
-#define GYRO_CONFIG_USE_GYRO_BOTH   2
+typedef enum {
+    GYRO_CONFIG_USE_GYRO_1,
+    GYRO_CONFIG_USE_GYRO_2,
+    GYRO_CONFIG_USE_GYRO_3,
+    GYRO_CONFIG_USE_GYRO_ALL,
```

Comment:
See above - this will break MSP. And it will break again in fourth gyro is added.